applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -332,9 +332,15 @@ static void syncable_cb(struct bt_audio_broadcast_sink *sink, bool encrypted)
 	strncpy(bis_encryption_key, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY, 16);
 #endif /* (CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED) */
 
-	if (active_stream.stream->ep->status.state == BT_AUDIO_EP_STATE_STREAMING ||
-	    !playing_state) {
-		LOG_DBG("Syncable received, but either in paused_state or already in a stream");
+	if (active_stream.stream->ep != NULL) {
```

Comment:
Looks great. Wondering though if this first if should be an error check if the endpoint is NULL. Can you comment @alexsven, @erikrobstad or @rick1082 ?

Reply:
@ladivin I started a new build for you.

Reply:
ep is NULL (at least) the first time syncable_cb is called, so I think the code is fine as is